Automattic/themes
# Hever: css variable approach to color annotations

#### Changes proposed in this Pull Request:

This PR implements color annotations for Hever using Seedlet's method. We are using css variables in an attempt to improve the performance of color annotations in general and to prepare this theme for Global Styles in the future.

![Screen Capture on 2021-01-14 at 15-45-07(4)](https://user-images.githubusercontent.com/3593343/104607857-8bbb2600-5681-11eb-85fe-cbb6b76677ca.gif)

To Do: 

- [x] Implement IE ponyfill to support css variables on that browser (in the same way this PR does: https://github.com/Automattic/themes/pull/2542)

#### Testing:

- Make sure that the colors are consistent between the customizer view, the frontend of the site, and the editor.
- The block color palette should reflect the changes made on the customizer (if we insert a button, the color options for Primary, Secondary, etc should match the ones we have customized for the theme instead of the default ones)
- Check that going back to the default palette has the desired effect.

#### Related issue(s):

#2991
